README.md

```
@@ -7,90 +7,86 @@
-## 📌 Navigation & Status
-
-| Section | Description | Version |
-| :--- | :--- | :--- |
-| [🚀 Quick Start](#-getting-started) | Get up and running in minutes | **Current: v1.0.2** |
-| [🎬 Demos](#-demos--showcases) | Visual previews and galleries | [Changelog](CHANGELOG.md) |
-| [⚙️ Architecture](#️-architecture) | System design and execution flow | [Statistics](STATS.md) |
-| [🛡️ Safety](#️-safety--security) | Security protocols and read-only mode | [Issues](ISSUES.md) |
+## Table of Contents
+
+- [Key Features](#key-features)
+- [Demos](#demos)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)